selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -353,9 +367,22 @@ public void waitItemIsNotPresentVisibleArea(String item) {
    */
   public void selectItem(String path) {
     String locator = "//div[@path='/" + path + "']/div";
+
     waitItem(path);
-    WebElement item = seleniumWebDriver.findElement(By.xpath(locator));
-    item.click();
+    try {
+      new WebDriverWait(seleniumWebDriver, EXPECTED_MESS_IN_CONSOLE_SEC)
+          .until(ExpectedConditions.visibilityOfElementLocated(By.xpath(locator)))
+          .click();
+    } catch (StaleElementReferenceException ex) {
+      LOG.error(ex.getLocalizedMessage(), ex);
```

Comment:
IMHO it's better to log exception as warn() here so as we don't interrupt the execution of tests.